evaluator.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+
+const githubUsername = process.env.GITHUB_ACTOR || 'no_actor';
+const githubRepositoryName = process.env.GITHUB_REPOSITORY || 'no_repository';
+
+// https://nodejs.org/en/knowledge/command-line/how-to-parse-command-line-arguments/
+const evaluationFileContent = fs.readFileSync(process.argv[2]);
+const testData = JSON.parse(evaluationFileContent);
+const results = testData.testResults[0].assertionResults;
+
+const mappingUniTestsToRequirements = fs.readFileSync(process.argv[3]);
+
+const evaluations = results.map((result) => {
+  const testName = result.fullName;
+  return {
+    requirement_id: mappingUniTestsToRequirements[testName],
+    grade: (result.status === 'passed') ? 3 : 1
+  }
+});
+
+fs.writeFileSync(process.argv[4], JSON.stringify({
```

Comment:
Why are you getting the write file from outside instead of create/write file with the `fs` lib itself?

Reply:
`process.argv[4]` contains the filename to save the jest results

Reply:
Ok, but why this need to be an `argv`? I didn't get the point of using an external input for that... :thinking:

Reply:
`process.argv[4]` is the way to get parameters from line command:
```bash
node test.js first-parameter second-parameter
```

Reply:
Actually, what I mean is: Why using `fs.writeFileSync(process.argv[4], ...)` instead of simply using something like `fs.writeFileSync('result.json', ...)`?

Reply:
Because I think `fs.writeFileSync(process.argv[4])` is more generic than `fs.writeFileSync('result.json')`.